### DISCUSSION TITLE
Import into list

### DISCUSSION THREAD

**Discussion by dcflachs** [Kovah/LinkAce#594] [Ideas & Feature Requests]
It would be really nice if imports could be done directly to a list.

**Comment by charlesroper** [comment]
Adding a +1 to this one. I think the only way to do this right now is import with a unique tag, then run an SQL query to update links with that tag?

**Comment by Kovah** [comment]
That's a great addition for importing. Together with bulk edits, this will definitely make imports easier to handle.
See #757

**Comment by charlesroper** [comment]
> Together with bulk edits, this will definitely make imports easier to handle.

Is bulk editing on the roadmap?

**Reply by Kovah** [reply]
It's basically done, but will ship with LinkAce 2 in the next months. I want to push this next step a bit more, instead of jugging between old new new versions.

**Reply by charlesroper** [reply]
Excellent, thank you very much. Looking forward to it. Bulk editing will be a very welcome addition.
